alerts.tf

```diff
@@ -11,7 +11,7 @@ module "div-bad-requests-alert" {
   frequency_in_minutes = 5
   time_window_in_minutes = 5
   severity_level = "2"
-  action_group_name = "Divorce Support"
+  action_group_name = "div-support"
```

Comment:
To match https://github.com/hmcts/div-shared-infrastructure/blob/master/action-groups.tf#L14